src/js/view/controls/components/volumetooltip.js

```diff
@@ -46,4 +46,9 @@ export default class VolumeTooltip extends Tooltip {
     toggleValue() {
         this.trigger('toggleValue');
     }
+
+    destroy() {
+        this.off();
```

Comment:
Not sure if you need this - where are the event listeners added? You should also remove the container listeners added in the constructor, plus call `this.ui.destroy()` if you want to clean-up everything nicely.

Reply:
@karimMourra requested i did this. Also, the volume tooltip is no longer constructed with `UI`

Reply:
What about the UI on line 37?

Reply:
Sorry, I mean the actual containing element needs the event listeners. The `UI` handles the actual slider but the `blur` and `focus` listeners are on the overlay container.

Reply:
no no, I asked you to remove the event listener that you are adding on the container.

Reply:
You should be removing the listener mentioned in this comment https://github.com/jwplayer/jwplayer/pull/3194/files#r238739450